### PR TITLE
Patch: Support for multiple Teams Accounts at the same time

### PR DESCRIPTION
**Pull request by Xathviar**
I'm always frustrated when I have to use my three different Teams Instances and managing them is quite difficult.
I usually have my team instances managed via teams-for-linux, one with my browser, and one with the incognito tab of my browser.
With the provided code one can run multiple teams-for-linux Instances at the same time and only has to create a different .desktop file for each of them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.0.49",
+  "version": "1.0.50",
```
Comment (IsmaelMartinez) [inline package.json]: another release has increased this to 50, so adding another number :D
```suggestion
  "version": "1.0.51",
```

**Review by IsmaelMartinez — APPROVED** [review]

**Comment by nneul** [comment]
I realize this has already been merged (THANK YOU!) -- is there any way you know of to be able to disambiguate the toolbar/status icons? I had been doing something similar to this just overriding $HOME - but this is much cleaner -- still has same issue with not being able to distinguish the toolbar icons.

**Comment by IsmaelMartinez** [comment]
Hi @nneul , you might be able to accomplish this with the `appIcon` option in https://github.com/IsmaelMartinez/teams-for-linux/tree/develop/app/config but you might need to provide a full path. That should just work.

**Comment by nneul** [comment]
@IsmaelMartinez  that worked great, thank you!

**Pull request merged**